src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -1188,14 +1188,18 @@ def _process_use_stmts(parent, nodes):
                     # will replace a previous import with an empty only-list.
                     pass
                 for name in decl.items[4].items:
-                    # The DataSymbol adds itself to the list of symbols
-                    # imported by the Container referenced in the
-                    # GlobalInterface.
                     sym_name = str(name).lower()
                     if sym_name not in parent.symbol_table:
+                        # We're dealing with a symbol named in a use statement
+                        # in the *current* scope therefore we don't need to
```

Comment:
s/don't need to check any/should not check/

Reply:
Done (although I went for "we do not check...").